core/modules/cms/store/block/getters.ts

```diff
@@ -1,14 +1,35 @@
 import { GetterTree } from 'vuex'
 import CmsBlockState from '../../types/CmsBlockState'
 import RootState from '@vue-storefront/core/types/RootState'
+import { Logger } from '@vue-storefront/core/lib/logger'
 
 const getters: GetterTree<CmsBlockState, RootState> = {
-  cmsBlocks: (state) => state.items,
-  cmsBlockIdentifier: (state) => (identifier) => {
-    return state.items.find(item => typeof item === 'object' && item.identifier === identifier)
+  cmsBlocks: (state, getters) => {
+    Logger.error('The getter cmsBlock/addMethod has been deprecated please change to cmsBlock/getCmsBlocks')()
+
+    return getters.getCmsBlocks
+  },
+  cmsBlockIdentifier: (state, getters) => (identifier) => {
+    Logger.error('The getter cmsBlock/cmsBlockIdentifier has been deprecated please change to cmsBlock/getCmsBlockIdentifier')()
+
+    return getters.cmsBlockIdentifier(identifier)
   },
-  cmsBlockId: (state) => (id) => {
-    return state.items.find(item => item.id === id)
+  cmsBlockId: (state, getters) => (id) => {
+    Logger.error('The getter cmsBlock/cmsBlockId has been deprecated please change to cmsBlock/getCmsBlockId')()
+
+    return getters.getCmsBlockId(id)
+  },
```

Comment:
Shouldn’t it be getCmsBlocById / I mean the adding “by” we will end up with improved grammar; well it’s just a proposition ;)